manifests/piped/templates/configmap.yaml

```diff
@@ -13,12 +13,12 @@ data:
     spec:
       projectID: quickstart
       pipedID: {{ .Values.quickstart.pipedId }}
-      pipedKeyFile: /etc/piped-secret/piped-key
+      pipedKeyData: {{ .Values.quickstart.pipedKeyData }}
       apiAddress: pipecd:8080
       syncInterval: 1m
       repositories:
         - repoId: examples
-          remote: {{ .Values.quickstart.gitRepoRemote }}
+          remote: https://github.com/pipe-cd/examples.git
```

Comment:
If we use this repository as the quickstart config, the users cannot try the git commit hooked deployments.
I think it's nice to place this value as the default value, not as the constant value.

Reply:
To make the setup more accessible, I'm thinking about only making the quickstart scenario like
- prepare k8s cluster
- install pipecd controllplane
- install piped (with minimal setting)
- add app and trigger quicksync as default

For other things, like trigger canary, it could be better to make it elsewhere. Also, we can refer to the docs, which show how to change the piped setting for adding other platform providers, adding other remote repos, etc.

Reply:
For example, we have this docs: https://pipecd.dev/docs-v0.48.x/user-guide/managing-piped/adding-a-git-repository/

Reply:
I got your point, and I noticed that we have [the tutorial](https://github.com/pipe-cd/tutorial), so we can try other remote repos with this.